h2o-docs/src/product/welcome.rst

```diff
@@ -1,1078 +1,144 @@
+Basic framework
+---------------
 
-H2O’s REST API allows access to all the capabilities of H2O from an external program or script via JSON over HTTP. The Rest API is used by H2O’s web interface (Flow UI), R binding (H2O-R), and Python binding (H2O-Python).
+H2O's core code is written in Java. A distributed key-value store is used to access and reference data, models, objects, etc. across all nodes and machines. The algorithms are implemented on top of H2O's distributed map-reduce framework and utilize the Java fork/join framework for multi-threading. The data is read in parallel and is distributed across the cluster. It is stored in-memory in a columnar format in a compressed way. H2O's data parser has built-in intelligence to guess the schema of the incoming dataset and supports data ingest from multiple sources in various formats.
 
-The speed, quality, ease-of-use, and model-deployment for the various cutting edge Supervised and Unsupervised algorithms like Deep Learning, Tree Ensembles, and GLRM make H2O a highly sought after API for big data data science.
+REST API
+~~~~~~~~
 
-H2O is licensed under the `Apache License, Version 2.0 <http://www.apache.org/licenses/LICENSE-2.0>`_.
+H2O's REST API allow access to all the capabilities of H2O frm an external program or script through JSON over HTTP. The REST API is used by H2O's web interface (Flow UI), R binding (H2O-R), and Python binding (H2O-Python).
 
-Requirements
-------------
-
-At a minimum, we recommend the following for compatibility with H2O:
-
--  **Operating Systems**:
-
-   -  Windows 7 or later
-   -  OS X 10.9 or later
-   -  Ubuntu 12.04
-   -  RHEL/CentOS 6 or later
-
--  **Languages**: R and Python are not required to use H2O unless you want to use H2O in those environments, but Java is always required (see `below <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/welcome.html#java-requirements>`__).
-
-   -  R version 3 or later
-   -  Python 3.6.x, 3.7.x, 3.8.x, 3.9.x, 3.10.x, 3.11.x 
-
--  **Browser**: An internet browser is required to use H2O's web UI, Flow. Supported versions include the latest version of Chrome, Firefox, Safari, or Internet Explorer.
-
-Java Requirements
-~~~~~~~~~~~~~~~~~
-
-H2O runs on Java. To build H2O or run H2O tests, the 64-bit JDK is required. To run the H2O binary using either the command line, R, or Python packages, only 64-bit JRE is required.
+The speed, quality, ease-of-use, and model-deployment for our various supervised and unsupervised algorithms (such as Deep Learning, GLRM, or our tree ensembles) make H2O a highly sought after API for big data data science.
```

Comment:
You list some algos here,  I would suggest you list all of them here to make people know all of our offerings:

adaboost
aggregator
anovaglm
coxph
deep learning
stacked ensemble
gam
glm
glrm
isotonic regression
kmeans
modelselection
naivebayes
pca
psvm
rulefit
isolation forest/extended isolation forest
uplift
word2vec